root/defaults/nginx/server-confs/authelia-server.conf.sample

```diff
@@ -1,8 +1,8 @@
-## Version 2021/05/28 - Changelog: https://github.com/linuxserver/docker-swag/commits/master/root/defaults/authelia-server.conf
+## Version 2021/05/28 - Changelog: https://github.com/linuxserver/docker-swag/commits/master/root/defaults/nginx/server-confs/authelia-server.conf.sample
 # Make sure that your authelia container is in the same user defined bridge network and is named authelia
 
 location ^~ /authelia {
-    include /config/nginx/proxy.conf;
+    include /config/nginx/location-confs/*.conf;
     include /config/nginx/resolver.conf;
```

Comment:
Can also be removed

Reply:
Resolved in https://github.com/linuxserver/docker-swag/pull/169/commits/daf99713a97f809748a67d6c3ec40ed1e1fcfe24